frontend/src/scenes/settings/user/PersonalAPIKeys.tsx

```diff
@@ -1,78 +1,231 @@
+                footer={
+                    <div>
+                        <div className="flex flex-1 gap-2 justify-end">
+                            <LemonButton type="secondary" onClick={() => setEditingKeyId(null)}>
+                                Cancel
+                            </LemonButton>
+
+                            <LemonButton
+                                type="primary"
+                                htmlType="submit"
+                                loading={isEditingKeySubmitting}
+                                disabled={!editingKeyChanged}
+                                onClick={() => submitEditingKey()}
+                            >
+                                {isNew ? 'Create key' : 'Save key'}
+                            </LemonButton>
+                        </div>
+                    </div>
+                }
+            >
+                <div className="space-y-2">
+                    <LemonField name="label" label="Label">
+                        <LemonInput placeholder='for example "Zapier"' maxLength={40} />
+                    </LemonField>
+
+                    <LemonField
+                        name="scoped_organizations"
+                        label="Organization access"
+                        info="Limit this API key to only perform operations on certain organizations."
+                    >
+                        <LemonSelectMultiple
+                            mode="multiple"
+                            data-attr="organizations"
+                            options={
+                                allOrganizations.map((org) => ({
+                                    key: `${org.id}`,
+                                    label: org.name,
+                                    labelComponent: (
+                                        <Tooltip
+                                            title={
+                                                <div>
+                                                    <div className="font-semibold">{org.name}</div>
+                                                    <div className="text-xs whitespace-nowrap">ID: {org.id}</div>
+                                                </div>
+                                            }
+                                        >
+                                            <span className="flex-1">{org.name}</span>
+                                        </Tooltip>
+                                    ),
+                                })) ?? []
                             }
-                        }}
+                            placeholder="All organizations"
+                        />
+                    </LemonField>
+
+                    <LemonField
+                        name="scoped_teams"
+                        label="Project access"
+                        info="Limit this API key to only perform operations on certain projects."
                     >
```

Comment:
Hmmm I see your point. Feels like the solution isn't obvious though. Checkboxes feels somehow way more cumbersome...

I'm personally tempted to just go with this as is for now (the fatigue of this PR is really setting in) and happily have a follow up contribution from yourself or anyone else. This does work, even if it isn't perfect.

Reply:
I don't think it quite works though, as in I really don't feel confident from the UI about how the scoping is structured ("does the key ultimately work on all projects within the organization, or only those selected"). So in my view this would be massively more obvious with a toggle "Scoping by <organization/project>", which would simply show either one mutliselect or the other – but not the two at once.